tests/queries/0_stateless/02011_tuple_vector_functions.sql

```diff
@@ -0,0 +1,58 @@
+SELECT tupleHammingDistance(tuple(1), tuple(1));
+SELECT tupleHammingDistance((1, 3), (1, 2));
```

Comment:
Also it's good to test errors for some corner cases, like number of elements doesn't match, empty tuples (is it possible now?), illegal types and so on.

Reply:
As `tupleHammingDistance` was not written by me, there are a lot of tests for this function in the file `01653_tuple_hamming_distance_2.sql`. I can add such corner cases checks with other functions.
To be honest, I do not know how to create an empty tuple (`tuple()` does not work).

Reply:
Added, you can check the description of the ['Corner cases' commit](7eb725a74358d97f1e3a27d4b78d917b13613b92).